CodePush.m

```diff
@@ -102,6 +102,17 @@ + (NSString *)getApplicationSupportDirectory
     return applicationSupportDirectory;
 }
 
++ (NSString *)getBinaryBundleDateString
```

Comment:
Isn't it typically Objective-C convention to leave off the `get` prefix for accessor methods?